test/python/pulse/test_transforms.py

```diff
@@ -267,7 +267,7 @@ def test_padding_schedule(self):
         )
 
         ref_sched = (
-            sched
+            sched  # pylint: disable=unsupported-binary-operation
```

Comment:
Annoyingly `py-version = "3.7"` is not enough to avoid needing different suppressions for different python versions in this file.